homeassistant/components/devolo_home_network/__init__.py

```diff
@@ -128,3 +175,18 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         hass.data[DOMAIN].pop(entry.entry_id)
 
     return unload_ok
+
+
+def require_reauth(hass: HomeAssistant, entry: ConfigEntry) -> None:
+    """Require the user to log in again."""
+    hass.add_job(
```

Comment:
This will be called from async context so we need to use an async api.

Use `hass.async_create_task`.